docs/data/charts/pie/pie.md

```diff
@@ -18,6 +18,14 @@ If you plan to update/reorder those data, you should add an `id` property which
 
 {{"demo": "BasicPie.js"}}
 
+## Donut chart
```

Comment:
Used "donut chart" as title as it is more common than "doughnut chart" according to Google trends. 

Still, normal text is still indexed by our search engine, so "doughnut chart" will also show a match.